src/Microsoft.ML.Data/Scorers/FeatureContributionCalculatingTransformer.cs

```diff
@@ -0,0 +1,673 @@
+   FeatureContributionCalculatingTransformer.FriendlyName, FeatureContributionCalculatingTransformer.LoaderSignature)]
+
+namespace Microsoft.ML.Runtime.Data
+{
+    /// <summary>
+    /// The FeatureContributionCalculationTransformer scores the model on an input dataset and
+    /// computes model-specific contribution scores for each feature.
+    /// </summary>
+    /// <remarks>
+    /// See the sample below for an example of how to compute feature importance using the FeatureContributionCalculatingTransformer.
+    /// </remarks>
+    /// <example>
+    /// <format type="text/markdown">
+    /// <![CDATA[
+    /// [!code-csharp[FCT](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/FeatureContributionCalculationTransform.cs)]
+    /// ]]>
+    /// </format>
+    /// </example>
+    public sealed class FeatureContributionCalculatingTransformer : RowToRowTransformerBase
+    {
+        // Apparently, loader signature is limited in length to 24 characters.
+        internal const string Summary = "For each data point, calculates the contribution of individual features to the model prediction.";
+        internal const string FriendlyName = "Feature Contribution Transform";
+        internal const string LoaderSignature = "FeatureContribution";
+
+        private readonly string _featureColumn;
+        private readonly BindableMapper _mapper;
+
+        private static VersionInfo GetVersionInfo()
+        {
+            return new VersionInfo(
+                modelSignature: "FCC TRAN",
+                verWrittenCur: 0x00010001, // Initial
+                verReadableCur: 0x00010001,
+                verWeCanReadBack: 0x00010001,
+                loaderSignature: LoaderSignature,
+                loaderAssemblyName: typeof(FeatureContributionCalculatingTransformer).Assembly.FullName);
+        }
+
+        /// <summary>
+        /// The Feature Contribution Calculation Transform scores the model on an input dataset and
+        /// computes model-specific contribution scores for each feature.
+        /// </summary>
+        /// <param name="env">The environment to use.</param>
+        /// <param name="predictor">A trained model that supports Feature Contribution Calculation, and which will be used for scoring.</param>
+        /// <param name="featureColumn">The name of the feature column that will be used as input.</param>
+        /// <param name="top">The number of top contributing features for each data sample that will be retained in the FeatureContribution column.</param>
+        /// <param name="bottom">The number of least contributing features for each data sample that will be retained in the FeatureContribution column.</param>
+        /// <param name="normalize">Whether the feature contributions should be normalized.</param>
+        /// <param name="stringify">Whether to output feature contributions in string key-value format.</param>
```

Comment:
>hether to output feature contributions in string key-value format. [](start = 37, length = 66)

might want to be a bit more specific: since the features get converted to numbers before the algorithms use them, if you want the contributions presented as string(key) - values set stringify to <langword>true</langword>